paddle/fluid/framework/ir/mkldnn/cpu_quantize_pass_tester.cc

```diff
@@ -588,6 +589,48 @@ TEST(CpuQuantizePass, elementwise_add_unsigned_and_signed_input) {
            expected_operators, added_nodes, 1.f, 1.f, "", "b");
 }
 
+static const std::initializer_list<std::string> variable_names_elementwise_mul =
```

Comment:
Here code for elementwise_add and elementwise_mul should be the same. So you can parametrize `BuildProgramDescElementwiseAdd()` to `BuildProgramDescElementwise(const std::string elementwise_type)`. It will simplify test for next elementwise ops. You can reuse similar solution that was added for immutable ops in `TestImmutableOpWithManyOutputs`